tests/functions/test_encode_categorical.py

```diff
@@ -24,110 +21,135 @@ def test_encode_categorical(df):
 @given(df=df_strategy())
 def test_encode_categorical_missing_column(df):
     """
-    Raise JanitorError for missing columns
+    Raise ValueError for missing columns
```

Comment:
Thank you for doing this! The `JanitorError` was getting overused and becoming uninformative. Getting rid of `JanitorError` is a great idea.